design/mappings/file_manifest_mappings.yml

```diff
@@ -0,0 +1,16 @@
+fileManifest:
+  Filename: asset.path
+  Filetype: [
+    asset.path, #Split and get file extension
+    asset.encodingFormat, #Follows application/<filetype> format but type is just string
+    dandiset.metadata.assetsSummary.dataStandard # has RRID + standard name but is at data collection level
+  ]
+  Sample ID: asset.path #Split to remove id from filename
+  Project: [
+    asset.wasGeneratedBy, #schemaKey=Project Not as reliable
+    dandiset.metadata.wasGeneratedBy, #schemaKey=Project More reliable
```

Comment:
this is correct. an asset can belong to multiple collections. so deriving project from collection is appropriate.